src/styles.css

```diff
@@ -1,4 +1,5 @@
-@import url("https://fonts.googleapis.com/css2?family=Caudex:ital,wght@0,400;0,700;1,400;1,700&display=swap");
+@import "@fontsource/caudex/latin.css";
```

Comment:
Use this import instead
import '@fontsource/caudex/400-italic.css';
import '@fontsource/caudex/700-italic.css';